common/src/main/java/io/homeassistant/companion/android/common/sensors/SensorReceiverBase.kt

```diff
@@ -284,7 +284,8 @@ abstract class SensorReceiverBase : BroadcastReceiver() {
             try {
                 success = integrationUseCase.updateSensors(enabledRegistrations.toTypedArray())
                 enabledRegistrations.forEach {
-                    sensorDao.updateLastSendState(it.uniqueId, it.state.toString())
+                    sensorDao.updateLastSentState(it.uniqueId, it.state.toString())
+                    sensorDao.updateLastSentIcon(it.uniqueId, it.icon)
```

Comment:
This should be done in a transaction to improve performance (wrap them in [another function with `@Transaction`](https://developer.android.com/reference/kotlin/androidx/room/Transaction))

Reply:
Oh, I just saw this part in your message:

> Edit: or using the regular update function and just change two properties, like failed registrations

OK, then I'll go with the single update method with two properties approach.

Reply:
Done in a25c396